core/tests/account_service_tests.rs

```diff
@@ -99,6 +99,17 @@ fn import_account_corrupted() {
     ));
 }
 
+#[test]
+fn import_account_corrupted_base64() {
+    let core = test_core();
+
+    base64::decode("clearlyabadaccountstring").unwrap();
+    assert!(matches!(
+        core.import_account("clearlyabadaccountstring"),
+        Err(Error::UiError(ImportError::AccountStringCorrupted))
+    ));
+}
+
```

Comment:
Unsure about the base64 decode line

Reply:
confirms that this is valid base64, and therefore that the failure happens after base64 decoding